plugins/MantisCoreFormatting/core/MantisMarkdown.php

```diff
@@ -233,4 +233,29 @@ private static function init() {
 		}
 		return static::$mantis_markdown;
 	}
+
+	/**
+	 * Fix line indent
```

Comment:
It would be useful to focus on why the code is doing what it is doing, rather than what it is doing which is readable via the code.  For example, "Add a space at the end of the text in order to fix blah blah blah" and possibly reference the bug that this workaround is fixing.